packages/xo-web/src/common/editable/index.js

```diff
@@ -343,70 +334,30 @@ export class Number extends Component {
 
 export class Select extends Editable {
   static propTypes = {
-    options: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
-      .isRequired,
     renderer: PropTypes.func,
   }
 
-  componentWillReceiveProps (props) {
-    if (
-      props.value !== this.props.value ||
-      props.options !== this.props.options
-    ) {
-      this.setState({
-        valueKey: findKey(props.options, option => option === props.value),
-      })
-    }
-  }
-
-  get value () {
-    return this.props.options[this.state.valueKey]
-  }
-
-  _onChange = event => {
-    this.setState({ valueKey: getEventValue(event) }, this._save)
+  get value() {
+    return this.props.value
   }
 
-  _optionToJsx = (option, key) => {
-    const { renderer } = this.props
-
-    return (
-      <option key={key} value={key}>
-        {renderer ? renderer(option) : option}
-      </option>
-    )
-  }
-
-  _onEditionMount = ref => {
-    // Seems to work in Google Chrome (not in Firefox)
-    ref && ref.dispatchEvent(new window.MouseEvent('mousedown'))
-  }
-
-  _renderDisplay () {
+  _renderDisplay() {
     const { children, renderer, value } = this.props
-
-    return children || <span>{renderer ? renderer(value) : value}</span>
-  }
-
-  _renderEdition () {
-    const { saving, valueKey } = this.state
-    const { options } = this.props
-
     return (
-      <select
-        autoFocus
-        className={classNames('form-control', styles.select)}
-        onBlur={this._closeEdition}
-        onChange={this._onChange}
-        onKeyDown={this._onKeyDown}
-        readOnly={saving}
-        ref={this._onEditionMount}
-        value={valueKey}
-      >
-        {map(options, this._optionToJsx)}
-      </select>
+      children || (
+        <span>{renderer !== undefined ? renderer(value) : value}</span>
+      )
     )
   }
+
+  _renderEdition = () => (
+    <FormSelect
+      {...this.props}
+      onBlur={this._closeEdition}
+      onKeyDown={this._onKeyDown}
+      openOnFocus
```

Comment:
This doesn't seem to work in `host/network`.

Reply:
it works but `Select in host\network` dosen't have an autoFocus.

Reply:
Then we should always pass `openOnFocus` and `autoFocus` here as we always want the select to open by itself when we enter edition mode.